macro-analytics/README.md

```diff
@@ -0,0 +1,18 @@
+
+# Build a Google Analytics dashboard 🖥️ with an Apps Script, a Sheets add-on,and Google Sites
+
+This is an Apps Script sample to automate the setup and summary of charts
+from Google Analytics into Google Sheets. Charts can optionally be inserted
```

Comment:
Nit: it's easier for diffs to have every sentences start on its own line

Reply:
Good call, added!